composer.json

```diff
@@ -20,7 +20,9 @@
     "require-dev": {
         "phpunit/phpunit": "^9.6",
         "squizlabs/php_codesniffer": "3.*",
-        "phpspec/prophecy-phpunit": "^2.1"
+        "phpspec/prophecy-phpunit": "^2.1",
+        "ext-protobuf": "*",
+        "ext-grpc": "*"
```

Comment:
without those two it seems to be impossible to run test suite, also it's anyway required by github workflows

Reply:
These test changes are really useful! Can we get them in a separate PR?

Reply:
do you prefer to keep extensions as required-dev or maybe detect if the extension is available and mark tests that rely on it as skipped?

Reply:
I think the latter provided a better user experience

Reply:
I think so too, will try to propose something later